include/libtorrent/settings_pack.hpp

```diff
@@ -244,7 +244,7 @@ namespace libtorrent
 			// ``send_redundant_have`` controls if have messages will be sent to
 			// peers that already have the piece. This is typically not necessary,
 			// but it might be necessary for collecting statistics in some cases.
-			// Default is false.
+			// Default is true.
```

Comment:
it may even be better to remove the mention of what the default is. the script that generates the documentation will include the default value already